docs/developer-guide.md

```diff
@@ -32,4 +32,13 @@ If you decide to make changes to this file, the following manual steps are requi
     - <kbd>Right-Click</kbd> > <kbd>Generate Parser Code</kbd>
     - <kbd>Right-Click</kbd> > <kbd>Generate JFlex Lexer</kbd>, in the file dialog click <kbd>Save</kbd> and <kbd>Replace</kbd>
 - `_DjinniLexer.flex`:
-    - <kbd>Right-Click</kbd> > <kbd>Run JFlex Generator</kbd>
\ No newline at end of file
+    - <kbd>Right-Click</kbd> > <kbd>Run JFlex Generator</kbd>
+  
+## Release process
+
+To release a new plugin version, the following steps must be followed:
+1. Create a new release-tag. Set a meaningful version number and describe what has changed in the new version.
+2. Wait. Github will automatically build the plugin, publish it to the [jetbrains marketplace](https://plugins.jetbrains.com/organization/cross-language-cpp) & upload the plugin-artifact to the release.
+3. The update will be verified by Jetbrains within a few days. Then it will be distributed to the users.
```

Comment:
Is this enough information? I guess you should do the first release after these changes, then I can be sure my documentation works.

Reply:
we will see when I play it through the next time, but this is a great starting point!